<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(75, 75, 75);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="GraphicsLayoutWidget" name="graph_spectre"/>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="GraphicsLayoutWidget" name="graph_rgb"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QSpinBox" name="sb_lowRed">
              <property name="maximumSize">
               <size>
                <width>55</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="toolTip">
               <string>Lower red limit</string>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 121, 121);
color: rgb(0, 0, 0);</string>
              </property>
              <property name="maximum">
               <number>1000</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRangeSlider" name="dSlider_red">
              <property name="maximum">
               <number>1024</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="sb_highRed">
              <property name="maximumSize">
               <size>
                <width>55</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="toolTip">
               <string>Upper red limit</string>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 121, 121);
color: rgb(0, 0, 0);</string>
              </property>
              <property name="maximum">
               <number>1000</number>
              </property>
              <property name="value">
               <number>341</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QSpinBox" name="sb_lowGreen">
              <property name="maximumSize">
               <size>
                <width>55</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="toolTip">
               <string>Lower green limit</string>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(125, 231, 106);
color: rgb(0, 0, 0);</string>
              </property>
              <property name="maximum">
               <number>1024</number>
              </property>
              <property name="value">
               <number>342</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRangeSlider" name="dSlider_green">
              <property name="maximum">
               <number>1024</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="sb_highGreen">
              <property name="maximumSize">
               <size>
                <width>55</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="toolTip">
               <string>Upper green limit</string>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(125, 231, 106);
color: rgb(0, 0, 0);</string>
              </property>
              <property name="maximum">
               <number>1024</number>
              </property>
              <property name="value">
               <number>682</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QSpinBox" name="sb_lowBlue">
              <property name="maximumSize">
               <size>
                <width>55</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="toolTip">
               <string>Lower blue limit</string>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(138, 173, 255);
color: rgb(0, 0, 0);</string>
              </property>
              <property name="maximum">
               <number>1024</number>
              </property>
              <property name="value">
               <number>683</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRangeSlider" name="dSlider_blue">
              <property name="maximum">
               <number>1024</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="sb_highBlue">
              <property name="maximumSize">
               <size>
                <width>55</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="toolTip">
               <string>Upper blue limit</string>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(138, 173, 255);
color: rgb(0, 0, 0);</string>
              </property>
              <property name="maximum">
               <number>1024</number>
              </property>
              <property name="value">
               <number>1024</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QComboBox" name="cmb_wave">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="toolTip">
             <string>Put the slider in wavenumber/wavelength</string>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(130, 130, 130);
</string>
            </property>
            <item>
             <property name="text">
              <string>WaveNumber</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>WaveLength</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QComboBox" name="cmb_set_maximum">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="toolTip">
               <string>Global Max = view intensity | Local Max = intensity doesn't count</string>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(130, 130, 130);</string>
              </property>
              <item>
               <property name="text">
                <string>Set to Global Maximum</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Set to Local Maximum</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pb_search">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="toolTip">
             <string>Search the folderpath</string>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(130, 130, 130);</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>Position :</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_x">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximumSize">
               <size>
                <width>25</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="toolTip">
               <string>X position</string>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(200, 200, 200);
color: rgb(0, 0, 0);</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_y">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximumSize">
               <size>
                <width>25</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="toolTip">
               <string>Y position</string>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(200, 200, 200);
color: rgb(0, 0, 0);</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>Laser :</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_laser">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>IBeamCursor</cursorShape>
              </property>
              <property name="toolTip">
               <string>Laser wavelength</string>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(200, 200, 200);
color: rgb(0, 0, 0);</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>883</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsLayoutWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QRangeSlider</class>
   <extends>QSlider</extends>
   <header>/gui/widgets/QRangeSlider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
